Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/UIKitElementTargeting.swift

```diff
@@ -32,6 +31,7 @@ internal class UIKitElementSelector: AppcuesElementSelector {
 
         super.init()
 
+        // note: accessibilityLabel is inhertied from base type NSObject and used here
```

Comment:
`inhertied`

Reply:
😅 and I even had my spell check on, doh. Thank you.